client/package.json

```diff
@@ -10,7 +10,7 @@
     "@bpmn-io/align-to-origin": "^0.7.0",
     "@bpmn-io/dmn-migrate": "^0.4.3",
     "@bpmn-io/extract-process-variables": "^0.5.1",
-    "@bpmn-io/form-js": "^0.7.2",
+    "@bpmn-io/form-js": "^0.8.0",
```

Comment:
Showing the errors in the properties panel is not yet supported, right? Do we have an issue with adding this feature to the linter (similar to BPMN)?

Sorry for asking now, and not in the linter integration PR!

Reply:
Mhh actually this is something I hadn't really considered. So we're talking about these if I understand correctly:

![image](https://user-images.githubusercontent.com/17801113/189824211-3d3fb6c1-24a6-4904-8c75-9402f4c1716c.png)

@philippfromme In your opinion, what kind of workload are we looking at in terms of adding these to the forms editor? From what I remember in `bpmn` this was a significant effort, no?

@christian-konrad Do you expect these to be in the form editor at this stage? Or are we happy to move on without them for now and would rather tackle these in the future?